app/controllers/api/v2/ansible_inventories_controller.rb

```diff
@@ -31,9 +31,34 @@ def hostgroups
         show_inventory :hostgroup_ids, :hostgroup_id
       end
 
+      api :POST, '/ansible_inventories/schedule_report',
```

Comment:
This sounds like a odd name for potential user, I would rather name it according to what it does, instead of how.
Maybe just `/ansible_inventories/schedule` would do?
Or `/ansible_inventories/schedule_generate` ?

Reply:
@ezr-ondrej, thanks, I think you're right, so I've changed it to `/ansible_inventories/schedule`.